tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -721,7 +721,7 @@ def test_tf_keras_autolog_model_can_load_from_artifact(tf_keras_random_data_run,
     client = MlflowClient()
     artifacts = client.list_artifacts(run.info.run_id)
     artifacts = (x.path for x in artifacts)
-    assert "model" in artifacts
```

Comment:
I think we can just remove this line.

Reply:
Also for other `assert "model" in artifacts` assertions? Is it enough to validate that the model can be loaded through `load_model("runs/:<id>/model")`?

Reply:
yep :)